src/server/service/kv.rs

```diff
@@ -112,11 +117,31 @@ impl<T: RaftStoreRouter + 'static, E: Engine, L: LockManager> Service<T, E, L> {
         let status = RpcStatus::new(code, Some(format!("{}", err)));
         ctx.spawn(sink.fail(status).map_err(|_| ()));
     }
+
+    fn check_common_names(&self, ctx: &RpcContext) -> bool {
```

Comment:
Please document this function. What are common names? What is common name checking? When does it need to be called? etc.

Reply:
Done